src/components/home/home.scss

```diff
@@ -3,6 +3,9 @@ $bg-img: url('../../images/meal-prep.jpg');
 #hero-block {
   background-image: $bg-img;
   background-size: cover;
+  display: block;
+  width: 100%;
+  height: 100%;
 
   .cta-block {
     background-color: #ffffffd8;
```

Comment:
This .cta-block is different in the mockup. The layers are laid on top of each other like this:
first, the hero block with the background image of the meals.
second, the white, transparent overlay background of the .cta block.